gpii/node_modules/gpii-db-operation/src/DbUtils.js

```diff
@@ -36,3 +36,40 @@ gpii.dbOperation.composeError = function (error, termMap) {
 gpii.dbOperation.getCurrentTimestamp = function () {
     return new Date().toISOString();
 };
+
+/**
+ * This function looks at an item to make sure it exists properly, useful for
+ * looking at the results of a `{dataStore}.findById` or similar operation. If
+ * the item exists as it should, a promise resolved with the item is
+ * returned, otherwise a rejected promise is returned with the supplied `errorMessage`.
+ *
+ * The current criteria for validity is that the item is not undefined and has
+ * the `type` that is supplied in the arguments.
+ *
+ * @param {Object} item - Item, mostly a return from another API or CouchDB call
+ * that we are checking to make sure it's of the appropriate type.
+ * @param {String} type - The `type` that the item should be of. This is to
+ * follow the convention we have for CouchDB records where each record has a `type`
+ * member.
+ * @param {String} errorMessage - If this isn't the appropriate type, this message will be
+ * included in the promise rejection payload.
+ * @param {String} errorCode - If this isn't the appropriate type, this error code will be
+ * included in the promise rejection payload.
+ * @return {fluid.promise} Promise, when the item is valid it resolves with the
```

Comment:
Missing coma after "when the item is valid".

Reply:
updated